### PR TITLE
Rename normalized_target_value to normalize_target_value in the Hoeffding Tree regressor.

### PR DESCRIPTION
**Pull request by AlbandeCrevoisier**
Changes proposed in this pull request:

* Rename the `normalized_target_value` method in `normalize_target_value`.

---

Checklist

- [x] Code complies with PEP-8 and is consistent with the framework.
- [x] Code is properly documented.
- [x] Tests are included for new functionality or updated accordingly.
- [x] Travis CI build passes with no errors.
- [x] Test Coverage is maintained (threshold is -0.2%).
- [x] Files changed (update, add, delete) are in the PR's scope (no extra files are included).

### CODE REVIEW

File: src/skmultiflow/trees/nodes/active_learning_node_adaptive_multi_target.py [hunk 1]
```diff
@@ -50,10 +50,10 @@ def update_weights(self, X, y, learning_ratio, rht):
         normalized_sample = rht.normalize_sample(X)
         normalized_pred = self.predict(normalized_sample)
 
-        normalized_target_value = rht.normalized_target_value(y)
+        normalize_target_value = rht.normalize_target_value(y)
```
Comment (jacobmontiel) [inline src/skmultiflow/trees/nodes/active_learning_node_adaptive_multi_target.py]: The method name change is correct, but the variable name should not be changed. The original names were overlapping which is not a good practice.

File: src/skmultiflow/trees/nodes/active_learning_node_perceptron.py [hunk 2]
```diff
@@ -102,9 +102,9 @@ def update_weights(self, X, y, learning_ratio, rht):
         """
         normalized_sample = rht.normalize_sample(X)
         normalized_pred = self.predict(normalized_sample)
-        normalized_target_value = rht.normalized_target_value(y)
+        normalize_target_value = rht.normalize_target_value(y)
```
Comment (jacobmontiel) [inline src/skmultiflow/trees/nodes/active_learning_node_perceptron.py]: The method name change is correct, but the variable name should not be changed. The original names were overlapping which is not a good practice.

File: src/skmultiflow/trees/nodes/active_learning_node_perceptron_multi_target.py [hunk 3]
```diff
@@ -110,10 +110,10 @@ def update_weights(self, X, y, learning_ratio, rht):
         normalized_sample = rht.normalize_sample(X)
         normalized_pred = self.predict(normalized_sample)
 
-        normalized_target_value = rht.normalized_target_value(y)
+        normalize_target_value = rht.normalize_target_value(y)
```
Comment (jacobmontiel) [inline src/skmultiflow/trees/nodes/active_learning_node_perceptron_multi_target.py]: The method name change is correct, but the variable name should not be changed. The original names were overlapping which is not a good practice.

File: src/skmultiflow/trees/nodes/inactive_learning_node_adaptive_multi_target.py [hunk 4]
```diff
@@ -50,9 +50,9 @@ def update_weights(self, X, y, learning_ratio, rht):
         normalized_sample = rht.normalize_sample(X)
         normalized_pred = self.predict(normalized_sample)
 
-        normalized_target_value = rht.normalized_target_value(y)
+        normalize_target_value = rht.normalize_target_value(y)
```
Comment (jacobmontiel) [inline src/skmultiflow/trees/nodes/inactive_learning_node_adaptive_multi_target.py]: The method name change is correct, but the variable name should not be changed. The original names were overlapping which is not a good practice.

File: src/skmultiflow/trees/nodes/inactive_learning_node_perceptron.py [hunk 5]
```diff
@@ -81,9 +81,9 @@ def update_weights(self, X, y, learning_ratio, ht):
         """
         normalized_sample = ht.normalize_sample(X)
         normalized_pred = self.predict(normalized_sample)
-        normalized_target_value = ht.normalized_target_value(y)
+        normalize_target_value = ht.normalize_target_value(y)
```
Comment (jacobmontiel) [inline src/skmultiflow/trees/nodes/inactive_learning_node_perceptron.py]: The method name change is correct, but the variable name should not be changed. The original names were overlapping which is not a good practice.

File: src/skmultiflow/trees/nodes/inactive_learning_node_perceptron_multi_target.py [hunk 6]
```diff
@@ -96,9 +96,9 @@ def update_weights(self, X, y, learning_ratio, rht):
         normalized_sample = rht.normalize_sample(X)
         normalized_pred = self.predict(normalized_sample)
 
-        normalized_target_value = rht.normalized_target_value(y)
+        normalize_target_value = rht.normalize_target_value(y)
```
Comment (jacobmontiel) [inline src/skmultiflow/trees/nodes/inactive_learning_node_perceptron_multi_target.py]: The method name change is correct, but the variable name should not be changed. The original names were overlapping which is not a good practice.

File: src/skmultiflow/trees/nodes/sst_active_learning_node.py [hunk 7]
```diff
@@ -111,18 +111,18 @@ def update_weights(self, X, y, learning_ratio, rht):
         normalized_sample = rht.normalize_sample(X)
         normalized_base_pred = self._predict_base(normalized_sample)
 
-        normalized_target_value = rht.normalized_target_value(y)
+        normalize_target_value = rht.normalize_target_value(y)
```
Comment (jacobmontiel) [inline src/skmultiflow/trees/nodes/sst_active_learning_node.py]: The method name change is correct, but the variable name should not be changed. The original names were overlapping which is not a good practice.

File: src/skmultiflow/trees/nodes/sst_active_learning_node_adaptive.py [hunk 8]
```diff
@@ -57,18 +57,18 @@ def update_weights(self, X, y, learning_ratio, rht):
         _, n_features = get_dimensions(X)
         _, n_targets = get_dimensions(y)
 
-        normalized_target_value = rht.normalized_target_value(y)
+        normalize_target_value = rht.normalize_target_value(y)
```
Comment (jacobmontiel) [inline src/skmultiflow/trees/nodes/sst_active_learning_node_adaptive.py]: The method name change is correct, but the variable name should not be changed. The original names were overlapping which is not a good practice.

File: src/skmultiflow/trees/nodes/sst_inactive_learning_node.py [hunk 9]
```diff
@@ -81,10 +81,10 @@ def update_weights(self, X, y, learning_ratio, rht):
         _, n_features = get_dimensions(X)
         _, n_targets = get_dimensions(y)
 
-        normalized_target_value = rht.normalized_target_value(y)
+        normalize_target_value = rht.normalize_target_value(y)
```
Comment (jacobmontiel) [inline src/skmultiflow/trees/nodes/sst_inactive_learning_node.py]: The method name change is correct, but the variable name should not be changed. The original names were overlapping which is not a good practice.

File: src/skmultiflow/trees/nodes/sst_inactive_learning_node_adaptive.py [hunk 10]
```diff
@@ -60,18 +60,18 @@ def update_weights(self, X, y, learning_ratio, rht):
         _, n_features = get_dimensions(X)
         _, n_targets = get_dimensions(y)
 
-        normalized_target_value = rht.normalized_target_value(y)
+        normalize_target_value = rht.normalize_target_value(y)
```
Comment (jacobmontiel) [inline src/skmultiflow/trees/nodes/sst_inactive_learning_node_adaptive.py]: The method name change is correct, but the variable name should not be changed. The original names were overlapping which is not a good practice.

File: src/skmultiflow/trees/nodes/active_learning_node_adaptive_multi_target.py (hunk 1 above)
Comment (AlbandeCrevoisier) [reply src/skmultiflow/trees/nodes/active_learning_node_adaptive_multi_target.py]: Indeed, my `sed` was slightly rushed.
I restored the name of the variable, it should be ok now.

**Comment by jacobmontiel** [comment]
Thanks for the changes @AlbandeCrevoisier 
The PR is ready to merge.

**Comment by jacobmontiel** [comment]
`Codecov` is reporting an error for the "patch" coverage. It seems that codecov can not find some files in the reference commit. The change does not add new code so this error is invalid and can be ignored.

**Pull request merged**